appveyor.yml

```diff
@@ -21,7 +40,7 @@ environment:
       CONDA_NPY: "19"
       WP_URL: 'http://heanet.dl.sourceforge.net/project/winpython/WinPython_2.7/2.7.10.3/WinPython-32bit-2.7.10.3.exe'
       WP_CRC: '99E825BCDE4FFA57E612A0D31DF1487861C96D1903E4D20C5843FA2651C9E33B'
-      DEPS: "numpy scipy matplotlib traits traitsui ipython h5py sympy scikit-learn dill mock"
+      DEPS: "numpy scipy matplotlib traits traitsui ipython h5py sympy scikit-learn dill mock natsort"
```

Comment:
TBH, you can probably just delete the 2.7 entries. I set this up before there was a decision to not have a code base that works on both versions of python.